crates/tabby-common/src/lib.rs

```diff
@@ -34,6 +33,10 @@ pub struct SourceFile {
 }
 
 impl SourceFile {
+    pub fn create_file_id(git_url: &str, filepath: &str) -> String {
+        format!("{}:{}", git_url, filepath)
+    }
+
```

Comment:
Since `git_url` is no longer part of SourceFile, move the function to `index.rs`